webapplication/indexing/processing/article3d.js

```
@@ -157,6 +157,24 @@ module.exports = async () => {
     }
   }
 
+  //Delete all 3d articles before indexing them - this removes the arcticles that have been deleted, from the index
+  let esDeletionQueryResult;
+  try {
+    esDeletionQueryResult = await es.deleteByQuery({
+      index: config.es.assetIndex,
+      body: {
+        query: {
+          match: {
+            file_format: 'Article/3D',
+          }
+        }
+      }
+    });
+  } catch(error) {
+    console.log("failed to find and delete all the 3d articles", error);

```

Hmm I can't decide if we should fail hard here and refuse to complete the indexing? Dunno...

---

Me neither